controllers/seedimage_controller.go

```diff
@@ -695,6 +699,69 @@ func defaultRawInitContainers(seedImg *elementalv1.SeedImage, buildImg string, p
 	}
 }
 
+func crossIsoInitContainers(seedImg *elementalv1.SeedImage, buildImg string, pullPolicy corev1.PullPolicy) []corev1.Container {
```

Comment:
This method is almost identical to `defaultIsoInitContainers` most of the lines are duplicates. Wouldn't it be easier to just parametrize it have a single method? Afterall looks like it would be enough to parametrize the creation of the initcontainer based on the target platform

Reply:
That's true, I was pretty sure it was gonna touch more code, but I will try to rewrite it in the `defaultIsoInitContainers` instead!